### ISSUE TITLE
[BUG]: Unexpected Error

### ISSUE THREAD

**Issue by marcwoons** [mandarons/icloud-docker#204]
I created the config.yaml file (only changed the username to my icloud), out it in the right folder (i hope), but can't get the container started with an unexpected error.

Here's what I get:

docker run
  -d
  --name='icloud-drive-sync'
  --net='bridge'
  --privileged=true
  -e TZ="America/Denver"
  -e HOST_OS="Unraid"
  -e HOST_HOSTNAME="TATP"
  -e HOST_CONTAINERNAME="icloud-drive-sync"
  -e 'PUID'='99'
  -e 'PGID'='101'
  -l net.unraid.docker.managed=dockerman
  -l net.unraid.docker.icon='https://help.apple.com/assets/60AD31069883FC55AC222539/60AD310B9883FC55AC22254A/de_DE/712e44cf3701cf5bc9580c9367fa5526.png'
  -v '/mnt/user/appdata/icloud-drive-sync/config.yaml':'/app/config.yaml':'rw'
  -v '/mnt/user/data/icloud-drive-sync/':'/app/icloud/':'rw'
  --restart always 'mandarons/icloud-drive'
71025e781f975dfba87c09b1b53eab91419caa4d835d78d222071d1e8f187b40
docker: Error response from daemon: failed to create task for container: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: error during container init: error mounting "/mnt/user/appdata/icloud-drive-sync/config.yaml" to rootfs at "/app/config.yaml": mount /mnt/user/appdata/icloud-drive-sync/config.yaml:/app/config.yaml (via /proc/self/fd/6), flags: 0x5000: not a directory: unknown: Are you trying to mount a directory onto a file (or vice-versa)? Check if the specified host path exists and is the expected type.

The command failed.

**Comment by mandarons** [comment]
Duplicate of #200 . Please check if you  have created a valid `/mnt/user/appdata/icloud-drive-sync/config.yaml` before starting the container.
